_posts/en/newsletters/2024-06-07-newsletter.md

```diff
@@ -0,0 +1,211 @@
+    receivers may not want to fallback to one of the base address types
+    (base58check, bech32, or bech32m), such as privacy-minded users of
+    silent payments.  The proposed update allows the body field to be
+    empty.
+
+  - *New query parameters:* the update describes three new keys:
+    `lightning` for [BOLT11][] invoices (currently in use), `lno` for LN
+    offers (proposed), and `sp` for silent payments (propsed).  It also
+    describes how the keys for later parameters should be named.
+
+  Corallo notes in his post that the changes are safe for all known
+  deployed software as wallets will ignore or reject any `bitcoin:` URIs
+  that they cannot successfully parse.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Core Lightning 24.05rc2][] is a release candidate for the next major
+  version of this popular LN node implementation.
+
+- Bitcoin Core 27.1rc1 FIXME:harding
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+FIXME:Gustavojfe
+
+- [Core Lightning #7252][] lightningd: don't ignore fee limits on mutual close.
+
+- [LDK #2931][] log-outbound-channels
+
+- [Rust Bitcoin #2644][] Add HMAC Extract-and-Expand Key Derivation Function
+
+- [BIPs #1541][] BIP431: Opt In Topologically Restricted Until Confirmation Transactions For More Robust Fee-bumping
+
+- [BIPs #1556][] BIP 337: Compressed Transactions
+
+- [BLIPs #32][] adds [BLIP32][] describing how proposed DNS-based
+  human-readable Bitcoin payment instructions (see [Newsletter
+  #290][news290 omdns]) can be used with [onion messages][topic onion
+  messages] to allow payments to be sent to an address like
+  `bob@example.com`.  For example, Alice instructs her LN client to pay
+  Bob.  Her client may not be able to securely resolve DNS addresses
+  directly, but it can use an onion message to contact one of its peers
+  that provides a resolution service.  The peer retrieves the DNS TXT
+  record for the `bob` entry at `example.com` and places the results
+  along with a [DNSSEC][] signature into an onion message reply to
+  Alice.  Alice verifies the information and uses it to request an
+  invoice from Bob using the [offers][topic offers] protocol.
+
+{% assign four_days_after_posting = page.date | date: "%s" | plus: 345600 | date: "%Y-%m-%d 14:30" %}
```

Comment:
Preview site is showing June 5th here which doesnt seem to jibe with the time the preview generation was made. (sorry I dont have the bandwidth to troubleshoot this)